Mathlib/Topology/Instances/CantorSet.lean

```diff
@@ -73,10 +73,45 @@ lemma zero_mem_preCantorSet (n : ℕ) : 0 ∈ preCantorSet n := by
 
 theorem zero_mem_cantorSet : 0 ∈ cantorSet := by simp [cantorSet, zero_mem_preCantorSet]
 
+theorem preCantorSet_antitone : Antitone preCantorSet := by
+  apply antitone_nat_of_succ_le
+  intro m
+  simp only [Set.le_eq_subset, preCantorSet_succ, Set.union_subset_iff]
+  induction m with
+  | zero =>
+    simp only [preCantorSet_zero]
+    constructor <;> intro x <;> simp only [Set.mem_image, Set.mem_Icc, forall_exists_index,
+      and_imp] <;> intro y _ _ _ <;> constructor <;> linarith
```

Comment:
I tried and didn't find an obvious golf. Maybe `grind` can do this in the future (right now, it can't).
I would perhaps write this as
```suggestion
    constructor <;> intro x <;>
      simp only [Set.mem_image, Set.mem_Icc, forall_exists_index, and_imp] <;>
      intro y _ _ _ <;> constructor <;> linarith